Continuación del desarrollo de la clase Trabajador

luistf24 opened the pull request.

luistf24 committed:
Servicio tiene que pertenecer a Servicio #5

Lo dejo comentado para que veamos a continuación que hacer con ello

luistf24 committed:
feat: Ampliamos clase trabjador closes #5

Eliminamos el struct Servicio ya que existe una Servicio que lo
modeliza y es la que se está usando para otras clases como informes.
Añadimos atributo nTrabajador que sevirá para identificar un trabajador
de otro y se ha comentado en #5

JJ commented on src/trabajador.ts:
```diff
@@ -1,21 +1,12 @@
-  peinado = "Peinado",
-  ondulacion = "Ondulacion",
-  extensiones = "Extensiones"
-}
```
En general, no suele ser una buena práctica borrar código. Es mejor dejarlo, y abrir un issue para refactorizar.

JJ commented on src/trabajador.ts:
```diff
@@ -1,21 +1,12 @@
   private servicios: Servicio[];
 
-  constructor(servicios: Servicio[]){
+  constructor(nTrabajador: number,servicios: Servicio[]){
```
nit: generalmente es mejor usar nombres completos: número de trabajadores, por ejemplo.
También, number es un número real, no entero

JJ reviewed (changes requested):
Algunas modificaciones, si no os importa

luistf24 committed:
Fix #5 revertimos cambios para evitar conflictos

Se ha hablado com otro grupo y para evitar los conflictos con los
cambios que ellos han realizado respecto el enum Servicios

luistf24 commented:
Este commit f416848 lo ha hecho @fjromeero
